corehq/apps/reports/forms.py

```diff
@@ -120,6 +120,12 @@ class ScheduledReportForm(forms.Form):
         coerce=int,
         choices=ReportNotification.hour_choices())
 
+    start_date = forms.DateField(
+        label='Report Start Date',
```

Comment:
translate?

Reply:
Can I translate all the labels in `ScheduledReportForm`? Or only "Report Start Date"?

Reply:
Also, I may have asked this before, but how do I know when it's best to translate a string immediately versus using `ugettext_lazy`?

Reply:
You can translate all the labels. As for the rationale for different translation functions, check this out: https://commcarehq.readthedocs.io/en/latest/translations.html